docs/sources/tempo/introduction/telemetry.md

```diff
@@ -0,0 +1,68 @@
+---
+aliases:
+  - /docs/tempo/getting-started/traces
+  - /docs/tempo/traces
+description: "Traces and telemetry"
+keywords:
+  - Grafana
+  - traces
+  - tracing
+title: Traces and telemetry
+weight: 200
+---
+
+# Traces and telemetry
+
+Metrics, logs, traces, and profiles form the pillars of observability.
+Correlating between the four pillars of observability helps create a full holistic view of your application and infrastructure.
+
+![The four pillars of observability](/media/docs/tempo/intro/four-pillars-observe.png.png)
+
+## Metrics
+
+Metrics provide the health data about the state of a system.
+Often, metrics are the first alert that something is going on and are where discovery first starts.
```

Comment:
```suggestion
Because they are numeric values and therefore can easily be compared against known thresholds, metrics are the foundation of alerts, which constantly run in the background and trigger when a value is outside of an expected range. This is typically the first sign that something is going on and are where discovery first starts.
```